extension/api.js

```diff
@@ -83,10 +83,22 @@
 			url += 'api/v3/notifications';
 		}
 
+		url += '?';
 		url += participating;
+		url += '&per_page=1';
```

Comment:
If `participating` is empty, the ampersand should not be there.

You can make the URL query args into an array and then join on `&` to make it easy to do:

```
query = []
if(participating){
  query.push(participating);
}
query.push('per_page=1');
url += query.join('&')
```